Jenkinsfile

```diff
@@ -8,7 +8,10 @@ def bc_section = 'build-configs'
 def my_bc = null
 
 pipeline {
-    agent { label 'maven' }
+    agent { label 'maven-jdk11' }
+    environment {
+        JAVA_HOME="/usr/lib/jvm/java-11-openjdk"
+    }
```

Comment:
Are you sure this is wise to define JDK11 here with this scope. I think the scope of JDK11 is only relevant to the 'Build' stage.
The risk of increasing the scope of JDK11 to all stages is the impact on 'Function Test' stage. 'Function Test' stage depends on the system default JDK to be 8.

Reply:
FWIW, in https://github.com/Commonjava/indy/pull/1791 I think I've fixed this problem. There's a bit of overlap between that PR and this one, but I've tested it locally with JDK11 and the ftests work. I didn't change the agent name, but in the Jenkins server configuration I modified the maven agent to use the latest tag for its image, which includes JDK11 on that image. I'm not sure if I've messed something up with that, though...

Reply:
I think it is same to use a global env vs. wraping the mvn commands in withEnv. 
This pr add an argLine and I will rebase this to see if CI happy.